### PR TITLE
Update README for April 2022 Releases.md

### PR DESCRIPTION
**Pull request by jordynmackool**
Updating the information for the recent/ upcoming sections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,119 +110,113 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
...
-**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_java_suite/releases/tag/5.3.0)
+**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_server/releases/tag/3.0.0).
 
----
+--
 
-#### SDL JavaScript Suite: 1.5.0
-**Release Date**: October 27, 2021
+#### Manticore: 2.9.0
+**Release Date**: November 2021
```
Comment (theresalech) [inline README.md:149]:
```suggestion
**Release Date**: December 8, 2021
```

File: README.md [hunk 2]
```diff
@@ -110,119 +110,113 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
...
 
-**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_javascript_suite/releases/tag/1.5.0).
+**Contents**: Contents of this release can be tracked on the [here](https://github.com/smartdevicelink/manticore/projects/4).
```
Comment (theresalech) [inline README.md:151]:
```suggestion
**Contents**: Release information can be found [here](https://github.com/smartdevicelink/manticore/releases/tag/2.9.0).
```

File: README.md [hunk 3]
```diff
@@ -110,119 +110,113 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
 
 ## Recent Releases
 
-#### SDL Protocol Spec: 5.4.1
```
Comment (theresalech) [inline README.md:113]: Recent Releases should be in chronological order, so Protocol Spec 5.4.1 and RPC Spec 8.0.0 should be listed before SDL Core 8.1.0.

File: README.md [hunk 4]
```diff
@@ -110,119 +110,113 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
...
+**Target Release Date**: October 26, 2022
 
-**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_ios/projects/27).
+**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_core/projects/22).
 
 ---
 
-#### SDL Java Suite: 5.4.0
-**Target Release Date**: April 13, 2022
+#### SDL iOS: 7.3.3
```
Comment (theresalech) [inline README.md:177]:
```suggestion
#### SDL iOS: 7.4.0
```
We should plan for this to be a minor release as usual, and will update accordingly if we determine only a hotfix is needed.

File: README.md [hunk 5]
```diff
@@ -110,119 +110,113 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
...
+**Target Release Date**: June 29, 2022
 
-**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_java_suite/projects/21).
+**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_ios/projects/29).
```
Comment (theresalech) [inline README.md:180]: Project should also be updated to 7.4.

File: README.md [hunk 6]
```diff
@@ -110,119 +110,113 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
...
+--
 
----
-#### SDL Policy Server: 2.13.0
-**Target Release Date**: April 13, 2022
+#### SDL Protocol Spec: 5.4.2
+**Target Release Date**: October 26, 2022
 
-**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_server/projects/7).
+**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/protocol_spec/projects/7).
```
Comment (theresalech) [inline README.md:208]: Project should also be updated to 5.5

File: README.md [hunk 7]
```diff
@@ -110,119 +110,113 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
...
-**Target Release Date**: April 13, 2022
+**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_server/projects/8).
 
-**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_core/projects/21).
+--
 
----
-#### SDL Policy Server: 2.13.0
-**Target Release Date**: April 13, 2022
+#### SDL Protocol Spec: 5.4.2
```
Comment (theresalech) [inline README.md:205]:
```suggestion
#### SDL Protocol Spec: 5.5.0
```

We should plan for this to be a minor release as usual, and will update accordingly if we determine only a hotfix is needed.

File: README.md [hunk 8]
```diff
@@ -110,119 +110,113 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
...
+**Target Release Date**: October 26, 2022
 
-**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_server/projects/7).
+**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/protocol_spec/projects/7).
 
 ---
 
-#### Manticore: 2.10.0
```
Comment (theresalech) [inline README.md:220]: We need to keep Manticore 2.10 as an upcoming release.

File: README.md [hunk 9]
```diff
@@ -110,119 +110,113 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
...
-**Target Release**: May 2022
+#### SDL RPC Spec: 8.1.0
+**Target Release Date**: OOctober 26, 2022
```
Comment (theresalech) [inline README.md:213]:
```suggestion
**Target Release Date**: October 26, 2022
```

**Review by theresalech — CHANGES_REQUESTED** [review]
@jordynmackool I've left a few comments and suggested changes. Please let me know if you have any questions!

File: README.md (hunk 5 above)
Comment (jordynmackool) [reply README.md:180]: done

File: README.md (hunk 8 above)
Comment (jordynmackool) [reply README.md:220]: done.

File: README.md (hunk 6 above)
Comment (jordynmackool) [reply README.md:208]: done.

File: README.md (hunk 3 above)
Comment (jordynmackool) [reply README.md]: done.

File: README.md (hunk 8 above)
Comment (jordynmackool) [reply README.md:220]: added.

File: README.md [hunk 10]
```diff
@@ -115,114 +115,116 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
...
+**Release Date**: April 13, 2022
 
-**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_ios/releases/tag/7.3.0)
+**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_core/releases/tag/8.1.0).
 
 ---
 
-#### SDL Java Suite: 5.3.0
-**Release Date**: October 27, 2021
+#### SDL iOS: 7.3.2
```
Comment (theresalech) [inline README.md:141]:
```suggestion
#### SDL iOS: 7.4.0
```

File: README.md [hunk 11]
```diff
@@ -115,114 +115,116 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
...
+**Release Date**: April 13, 2022
 
-**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_java_suite/releases/tag/5.3.0)
+**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_ios/releases/tag/7.3.2).
```
Comment (theresalech) [inline README.md:144]:
```suggestion
**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_ios/releases/tag/7.4.0).
```

File: README.md [hunk 12]
```diff
@@ -115,114 +115,116 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
...
 
-**Contents**: Release information can be found [here](https://github.com/smartdevicelink/manticore/releases/tag/2.9.0). 
+**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_server/releases/tag/3.0.0).
 
----
 
 ## Upcoming Releases
 
+#### Manticore: 2.10.0
+**Target Release Date**: May 2022
```
Comment (theresalech) [inline README.md:172]:
```suggestion
```

File: README.md [hunk 13]
```diff
@@ -115,114 +115,116 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
...
+**Target Release Date**: October 26, 2022
 
-**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_java_suite/projects/21).
+**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_core/projects/22).
 
 ---
 
-#### SDL JavaScript Suite: 1.6.0
-**Target Release Date**: April 13, 2022
+#### SDL iOS: 7.4.0
```
Comment (theresalech) [inline README.md:185]:
```suggestion
#### SDL iOS: 7.5.0
```

File: README.md [hunk 14]
```diff
@@ -115,114 +115,116 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
...
 
-#### SDL iOS: 7.4.0
-**Target Release Date**: April 13, 2022
+**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/manticore/projects/5). 
 
-**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_ios/projects/27).
+--
 
----
+#### SDL Core: 8.2.0
```
Comment (theresalech) [inline README.md:177]: Core should be after app libraries, to keep in chronological order.

**Review by theresalech — CHANGES_REQUESTED** [review]
@jordynmackool a few more comments for you!

File: README.md [hunk 15]
```diff
@@ -115,114 +115,115 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
...
-**Contents**: Release information can be found [here](https://github.com/smartdevicelink/manticore/releases/tag/2.9.0). 
+**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_server/releases/tag/3.0.0).
 
----
 
 ## Upcoming Releases
 
+#### Manticore: 2.10.0
+**Target Release Date**: May 2022
+**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/manticore/projects/5).
```
Comment (theresalech) [inline README.md:172]:
```suggestion

**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/manticore/projects/5). 
```
I think I wrongly suggested to delete this line in my last review, sorry!

File: README.md [hunk 16]
```diff
@@ -115,114 +115,115 @@ You can see a detailed list of accepted and implemented proposals for upcoming v
...
 
 ## Upcoming Releases
 
+#### Manticore: 2.10.0
+**Target Release Date**: May 2022
+**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/manticore/projects/5). 
 
-#### SDL iOS: 7.4.0
-**Target Release Date**: April 13, 2022
+---
```
Comment (theresalech) [inline README.md:174]:
```suggestion

---
```
You need an extra break here so that the line shows up.

**Review by theresalech — CHANGES_REQUESTED** [review]
@jordynmackool this should be my final feedback!

**Review by theresalech — APPROVED** [review]

**Pull request merged**